src/xdocs/checks/whitespace/whitespaceafter.xml.template

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document xmlns="http://maven.apache.org/XDOC/2.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/XDOC/2.0 https://maven.apache.org/xsd/xdoc-2.0.xsd">
+  <head>
+    <title>WhitespaceAfter</title>
+  </head>
+  <body>
+    <section name="WhitespaceAfter">
+      <p>Since Checkstyle 3.0</p>
+      <subsection name="Description" id="Description">
+        <p>
+          Checks that a token is followed by whitespace, with the exception that it
+          does not check for whitespace after the semicolon of an empty for iterator. Use Check
+          <a href="emptyforiteratorpad.html#EmptyForIteratorPad">
+          EmptyForIteratorPad</a> to validate empty for iterators.
+        </p>
+      </subsection>
+
+      <subsection name="Properties" id="Properties">
+        <div class="wrapper">
+          <table>
+            <tr>
+              <th>name</th>
+              <th>description</th>
+              <th>type</th>
+              <th>default value</th>
+              <th>since</th>
+            </tr>
+            <tr>
+              <td>tokens</td>
+              <td>tokens to check</td>
+
+              <td>
+                subset of tokens
```

Comment:
Please create a ticket for generation of tokens, we should be able to reliably extract these from the check itself.

Reply:
Done at https://github.com/checkstyle/checkstyle/issues/13399